backend/test_auth_system.py

```
@@ -0,0 +1,343 @@
+        else:
+            raise Exception(f"Unsupported method: {method}")
+        
+        result = {
+            "status_code": response.status_code,
+            "expected": expected_status,
+            "success": response.status_code == expected_status
+        }
+        
+        try:
+            result["data"] = response.json()
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)